### ISSUE TITLE
Add "Edit Game Configuration Profile" option when Right-Clicking game from Home menu

### ISSUE THREAD

**Issue by Guitar-Smash** [xenia-manager/xenia-manager#80]
Sometimes when I use the "Xenia Settings" menu, I forget to change the profile from "Xenia Canary Profile" to the particular game I want to modify. 

I would like this New "Edit Game Configuration Profile" option to redirect to Xenia Settings and target the desired game by default.

![image](https://github.com/user-attachments/assets/088b9682-0057-48bf-92ab-9d5ab8fc5479)

**Comment by CuddleBear92** [comment]
I do agree this should be a thing and is a must frankly, to lower the amount of weird UI choices in the manager.


Also to build on it, a config system similar to RPCS3 where you have a default that is changing for all titles and only the specific changes the user wants to make is happening on a per game basis. Such a thing with how the configs work now... probably would require it to generate a new config file from both the default and the game specific changes each time one of them are updated.

**Comment by shazzaam7** [comment]
I've been thinking for a while about how to deal with this (Do I just remove the "default" options or leave them in here). My current solution would be when saving changes in these "default" configurations is to ask the user if he wants to update the game configuration files that use that specific version of Xenia.

I can't track what has been changed and what hasn't (It could be done, but I think it's really not worth it), so it's either on me to remove the "default" options from the UI entirely or to add a prompt asking the user if he wants to apply these new "defaults" to games that use that specific version of Xenia

**Comment by Guitar-Smash** [comment]
I have seen different implementations (RPCS3 vs Duckstation/PCSX2) for this problem:

RPCS3 will give you two choices when generating a new Game Configuration Profile:
- Create Custom Configuration from Global Settings
- Create Custom Configuration from Default Settings

The "Global Settings" option in this case would be the "Xenia ... Profile" in Xenia Settings. 

The "Default Settings" should probably just be a file stored somewhere in the Xenia Manager path that is used for reference only. It should not be seen on the UI by the user (this will prevent the user from tampering with the true base default settings on the UI).

Whenever you make a change to the "Global Settings" (Xenia . . . Profile), the changes there should only affect the games with no specific configuration profile.

. . .

Now when you look at Duckstation or PCSX2, it gets more complicated. When you generate a new game profile, many of the settings default to a value called "Use Global Settings". In this case, any changes you make to Global, the game profile will react to it and apply it to whatever game uses that value.

If you were to go this route, you would have to add an "additional" option to the game profile view, where they have a "Use Global Setting" option for every value (see pictures below).

This route is obviously more complicated, but you get the advantage of applying Global Settings to existing per-game profile values that select "Use Global Settings".

In conclusion, the RPCS3 method could be easier to implement, but the Duckstation/PCSX2 would probably be more robust.

![image](https://github.com/user-attachments/assets/c84660b8-c4fe-438c-8c06-e80a3a84753e)
![image](https://github.com/user-attachments/assets/6f4b95d0-8213-48cf-9664-8440be1e0e12)

**Comment by CuddleBear92** [comment]
while RPCS3 and PCSX2 both do a 3 tired settings system... how this manager is just making config text files and just pointing to Xenia to open the specific config file all causes issues with making a similar system.

BUT! When a new game is added and a new config is generated, it could take whatever the current default setting is and apply it. that way the user can atleast set the settings before importing games.
Also an button to copy all the default settings over would be cool too.

**Comment by shazzaam7** [comment]
> BUT! When a new game is added and a new config is generated, it could take whatever the current default setting is and apply it. that way the user can atleast set the settings before importing games.
This is how it currently works. When you add the game, it will copy whatever the current default settings are.

> Also an button to copy all the default settings over would be cool too.
This is a great idea honestly. It kinda fits the "Use Global Settings" that @Guitar-Smash recommended and it's both simple and efficient.

I'll think about this, but honestly, a button to copy all of the default settings seems like a way to go right now.

**Comment by shazzaam7** [comment]
https://github.com/user-attachments/assets/8d646811-54c3-4255-9237-135f827bd199

This is what I had in mind.
It will read the default profile (Checks what version of Xenia it uses) into the UI and the user needs to press "Save Changes" to actually save the settings.
